snippets/csharp/VS_Snippets_Winforms/System.Windows.Forms.HtmlWindow/CS/Form1.cs

```diff
@@ -307,14 +307,14 @@ private void SuppressScriptErrors()
 		{
 			if (webBrowser1.Document != null)
 			{
-                webBrowser1.Document.Window.Error += new HtmlElementErrorEventHandler(scriptWindow_Error);
+             webBrowser1.Document.Window.Error += new HtmlElementErrorEventHandler(scriptWindow_Error);
```

Comment:
From the corefx [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), that we've adopted, this should be indented four spaces:

```suggestion
               webBrowser1.Document.Window.Error += new HtmlElementErrorEventHandler(scriptWindow_Error);
```